app/buyers/views/buyers.py

```diff
@@ -270,59 +279,229 @@ def view_brief_responses(framework_slug, lot_slug, brief_id):
+        question_key_sequence = []
+        boolean_list_questions = []
+        csv_rows = []
+        brief = context['brief']
+
+        questions = self.get_questions(context['framework_slug'],
+                                       context['lot_slug'],
+                                       'legacy_output_brief_response')
+
+        # Build header row from manifest and add it to the list of rows
+        for question in questions:
+            question_key_sequence.append(question.id)
+            if question['type'] == 'boolean_list' and brief.get(question.id):
+                column_headings.extend(brief[question.id])
+                boolean_list_questions.append(question.id)
+            else:
+                column_headings.append(question.name)
+        csv_rows.append(column_headings)
+
+        # Add a row for each eligible response received
+        for brief_response in context['responses']:
+            if all_essentials_are_true(brief_response):
+                row = []
+                for key in question_key_sequence:
+                    if key in boolean_list_questions:
+                        row.extend(brief_response.get(key))
+                    else:
+                        row.append(brief_response.get(key))
+                csv_rows.append(row)
+
+        return Response(
+            csv_generator.iter_csv(csv_rows),
+            mimetype='text/csv',
+            headers={
+                "Content-Disposition": (
+                    "attachment;filename=responses-to-requirements-{}.csv"
+                ).format(brief['id']),
+                "Content-Type": "text/csv; header=present"
+            }
+        ), 200
 
-    if not is_brief_correct(brief, framework_slug, lot_slug, current_user.id):
-        abort(404)
+    def generate_ods(self, brief, responses):
+        doc = SpreadSheet()
+
+        doc.add_font(FontFace(name="Arial", fontfamily="Arial"))
+
+        doc.add_style("default", "table-cell", (
+            TableCellProperties(wrapoption="wrap", verticalalign="top"),
+            TextProperties(fontfamily="Arial", fontnameasian="Arial",
+                           fontnamecomplex="Arial", fontsize="11pt"),
+        ))
+
+        doc.add_style("header", "table-cell", (
+            TextProperties(fontfamily="Arial", fontnameasian="Arial",
+                           fontnamecomplex="Arial", fontsize="21pt",
+                           fontweight="bold"),
+        ))
+
+        doc.add_style("title", "table-cell", (
+            TableCellProperties(wrapoption="wrap", verticalalign="top"),
+            TextProperties(fontfamily="Arial", fontnameasian="Arial",
+                           fontnamecomplex="Arial", fontsize="16pt",
+                           fontweight="bold"),
+        ))
+
+        doc.add_style("subtitle", "table-cell", (
+            TextProperties(fontfamily="Arial", fontnameasian="Arial",
+                           fontnamecomplex="Arial", fontsize="16pt"),
+        ))
```

Comment:
Not important, but I believe the Crown owns the font, and embedding fonts in ODF is definitely a thing :) https://www.gov.uk/guidance/open-document-format-odf-guidance-for-uk-government/embed-fonts-in-odf-documents